packages/neutrino/src/neutrino.js

```diff
@@ -6,6 +6,7 @@ const Config = require('webpack-chain');
 const ora = require('ora');
 const merge = require('deepmerge');
 
+const cwd = process.cwd();
```

Comment:
I'd like to not use `process.cwd()` here, if possible, outside of the constructor. See line 16, which introduces `options.root`.